ios-showcase-template/deviceTrust/DeviceTrustViewController.swift

```diff
@@ -108,18 +95,16 @@ class DeviceTrustViewController: UITableViewController {
         // Fetches the appropriate note for the data source layout.
         let detection = self.deviceChecks[indexPath.row]
 
-        // set the properties for the cell
-        cell.textLabel?.text = detection.label
-
         // set the text colouring
-        if detection.detected {
+        if(detection.passed) {
+            cell.textLabel?.text = detection.name + " not detected"
+            cell.textLabel?.textColor = GREEN_COLOR
+            cell.imageView?.image = UIImage(named: "ic_detected_false")
 
+        } else {
+            cell.textLabel?.text = detection.name + " detected"
```

Comment:
This is incorrect for device lock. Currently when the check fails it says "Device lock detected" and when it passes it says "Device lock not detected".